src/encoded/schemas/structural_variant_sample.json

```
@@ -343,6 +343,52 @@
         "schema_version": {
             "default": "3"
         },
+        "technical_review": {
+            "title": "Technical Review",
+            "type": "object",
+            "notes": "We will store technical review notes in a separate field `technical_review_note` perhaps, to simplify UI logic.",
+            "properties": {
+                "call": {
+                    "title": "Technical Review Call",
+                    "type": "boolean",
+                    "description": "True if call, false if no call, or undefined"
```

Can we clarify this comment to make it sound less as though there are 3 options? Also, what exactly is a "no call"; I don't entirely get this terminology.
```suggestion
                    "description": "True if call, false if no call or undefined"
```

---

No Call as in [this](https://gyazo.com/4232a63bade9ffff8b591c2fd7bc3156). A value of `False` in this case is separate from if it was undefined or not present (no value).

---

Outdated/resolved.